k8s-configuration/stockanalyzer-configmap.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: stockanalyzer-configmap
+data:
+  symbols: "SPUP,BNRG,HIG.PRG,RILYT,GBARF,WINN,PWWBF,RHBAF,CLVR,UPLD,AJGFF,HLLY,MSIU,ALNKF,UCRD,MGNT,IMAQ,SWRBF,BAYRY,RVHCS,APTPF,LALFS,HIVE,FNLIF,TLIS,CRTO,ORGO,OZBKF,UNIKF,DSPHF,FRCLL,CLMB,NCXS,GXRFF,JL,AMSRF,BITC,TLSNY,TGMR,TZPC,ADDHY,CBVTF,GIGM,CIBR,AFSIM,GULDF,CSBF,LFMDP,ABMC,LVWR.WS,PDEX,BFS.PRE"
```

Comment:
why it's the part of k8s configuration. It looks like we should move it to `application.yaml`